ui/app/components/app/token-cell/token-cell.component.js

```diff
@@ -62,7 +61,7 @@ export default class TokenCell extends Component {
       sidebarOpen,
       currentCurrency,
       image,
-    } = props
+    } = this.props
```

Comment:
This was to make the VSCode ESLint plugin stop complaining about unused props.

The actual ESLint version we use has no trouble with this, but... this is cleaner anyway so :man_shrugging:

Reply:
Nevermind - I'll fix this separately